test/test_platform.jl

```diff
@@ -2,14 +2,17 @@
     @testset "Platform Info" begin
         @test length(cl.platforms()) == cl.num_platforms()
         for p in cl.platforms()
-            @test p != nothing
-            @test pointer(p) != C_NULL
-            for k in [:profile, :version, :name, :vendor, :extensions]
-                @test p[k] == cl.info(p, k)
+            @testset "Platform $(p[:profile])" begin
+                @test p != nothing
+                @test pointer(p) != C_NULL
+                for k in [:profile, :version, :name, :vendor, :extensions]
+                    @test p[k] == cl.info(p, k)
+                end
+                # ignore version tests until OpenCL 3 support is available
+                #v = cl.opencl_version(p)
+                #@test 1 <= v.major <= 2
+                #@test 0 <= v.minor <= 2
```

Comment:
Is there a reason why this code block is indented differently? It seems that it only has whitespace changes?

Reply:
I added the `@testset "Platform $(p[:profile])" begin` block to get a better log when a test fail.

Reply:
Yes, but I think the indentation was changed accidentally? The diff shows various lines of code that are identical, just shifted to the right. Can you please adjust that?

Reply:
Also, I think commenting the test on opencl_version isn't ideal. You can just adapt it and replace the upper bound by 3?

Reply:
Oh nevermind about the indentation. I just saw that you introduced a new testset.

Reply:
> Also, I think commenting the test on opencl_version isn't ideal. You can just adapt it and replace the upper bound by 3?

done